crates/datafusion/src/physical_plan/functions.rs

```diff
@@ -302,6 +304,7 @@ impl FromStr for BuiltinScalarFunction {
             "sha512" => BuiltinScalarFunction::SHA512,
             "split_part" => BuiltinScalarFunction::SplitPart,
             "starts_with" => BuiltinScalarFunction::StartsWith,
+            "ends_with" => BuiltinScalarFunction::EndsWith,
```

Comment:
endsWith is different to ends_with, (and also for startsWith). So supporting starts_with does not mean supporting startsWith. Is it possible to move this new addtion to ch_fns and makes it as endsWith?